accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -579,6 +617,153 @@ all occurrences of `S*` in `T` shall be replaced with `S`.  As a result, legacy
 types will never appear as type annotations in migrated libraries, nor will they
 appear in reified positions.
 
+### Exports
+
+If an unmigrated library re-exports a migrated library, the re-exported symbols
```

Comment:
Does "unmigrated" mean the same as "legacy", and  "migrated" - "opted-in"?
If so, it would be nice to use the same names.

Reply:
yes.  done.